make/update-tor-release

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+GIT_ROOT=${GIT_ROOT:-$(git rev-parse --show-toplevel)}
+
+bosh create release --force --name tor --dir ${GIT_ROOT}/test-assets/tor-boshrelease
```

Comment:
I wonder if it would make sense to create the tar file at (test) run time? Probably not in this PR, though